Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -469,7 +469,14 @@ void HotkeyScheduler::Run()
       }
 
       if (IsHotkey(HK_TOGGLE_DUMPTEXTURES))
-        Config::SetCurrent(Config::GFX_DUMP_TEXTURES, !Config::Get(Config::GFX_DUMP_TEXTURES));
+      {
+        const bool enable_dumping = !Config::Get(Config::GFX_DUMP_TEXTURES);
+        Config::SetCurrent(Config::GFX_DUMP_TEXTURES, enable_dumping);
```

Comment:
Not something I'm going to address in this PR, but while I'm thinking about it, should this and the other calls in HotkeyScheduler be to SetBaseOrCurrent instead of SetCurrent?

As it stands, any settings changed by hotkeys get reverted when emulation ends, which isn't what I think most users would expect.